parts/masterparams.t

```diff
@@ -109,7 +109,7 @@
       "type": "string"
     },
     "fqdnEndpointSuffix":{
-      "defaultValue": "%s.%s.cloudapp.azure.com",
+      "defaultValue": "cloudapp.azure.com",
```

Comment:
The fix is proven to be working.
only rpv1 faced this, because it uses the default value.